fe/fe-core/src/main/java/org/apache/doris/transaction/DatabaseTransactionMgr.java

```diff
@@ -425,7 +425,10 @@ public void preCommitTransaction2PC(List<Table> tableList, long transactionId,
 
         unprotectedPreCommitTransaction2PC(transactionState, errorReplicaIds, tableToPartition,
                 totalInvolvedBackends, db);
-        LOG.info("transaction:[{}] successfully pre-committed", transactionState);
```

Comment:
Can it be changed to debug level?

Reply:
OK, debug level should be OK, which will cause performance issue in normal case.